### DISCUSSION TITLE
Is it possible to remove the space between tabs?

### DISCUSSION THREAD

**Discussion by synic** [nanozuki/tabby.nvim#122] [Q&A]
I've tried a few things, but it doesn't look like it's possible. I'm posting here just to make sure that's the case. Can the space between the tabs be removed?

**Comment by nanozuki** [comment]
What config do you use? The customized config can do this.

**Reply by synic** [reply]
Just the default mentioned in the docs. I didn't see where it was adding the space, so didn't know what to remove

**Reply by nanozuki** [reply]
In the preset config, tabby.nvim cannot do this. You can use a customized way to configure the tabline. In [setup example](https://github.com/nanozuki/tabby.nvim#setup), the space is added by option `margin`.

**Reply by synic** [reply]
I am using that setup example, however, when I change `margin` to an empty string, it does not remove the spacing between tabs, it removes spacing between the tab name and the open indicator, tab number, and close button:

<img width="300" alt="image" src="https://github.com/nanozuki/tabby.nvim/assets/30906/419ddc9a-8e45-4bde-8b2d-cd2c43919c85">

**Reply by nanozuki** [reply]
Oh, you are right... I read the code; this is not controlled by "margin." There is no 'margin' or 'space' in tab labels. The space is just made by half of the preview tab's right separator and half of the next tab's separator.

If we don't consider the neighborhood tabs and still use these nerd font symbols, it cannot remove the spaces.
